### PR TITLE
Update navigation for identity

### PR DESCRIPTION
**Pull request by walaura**
## What does this change?
Update the identity dropdown menu to have **_new options_**! **_dividers_**! and  **_icons_**!

This pr is a joint effort between identity and dotcom as *gasp* it touches the mobile menu AND the dropdowns. I did my best to reuse existing code (like the icon in the mobile nav) but I'm afraid I had to touch quite a long list of files to make it all work.

screen shot 2018-06-06 at 5 06 30 pm

## What is the value of this and can you measure success?
Make the user experience cleaner by signposting everything inside MMA directly from here

## Screenshots
screen shot 2018-06-06 at 3 01 13 pm
screen shot 2018-06-06 at 3 01 19 pm

### CODE REVIEW

File: common/app/views/fragments/nav/userAccountDropdown.scala.html
Comment (SiAdcock) [inline common/app/views/fragments/nav/userAccountDropdown.scala.html]: What is this, XML? 🙀

Comment (walaura) [reply common/app/views/fragments/nav/userAccountDropdown.scala.html]: haha I'll tell myself if I leave `<hr>` open html5 style I'd be a hypocrite for using `</li>`

File: static/src/stylesheets/layout/nav/_top-bar-dropdown.scss
Comment (SiAdcock) [inline static/src/stylesheets/layout/nav/_top-bar-dropdown.scss]: Can we use a direct descendant selector here?

File: static/src/stylesheets/layout/nav/_top-bar-dropdown.scss
Comment (SiAdcock) [inline static/src/stylesheets/layout/nav/_top-bar-dropdown.scss]: And maybe direct descendant here!

**Review by SiAdcock — APPROVED** [review]
Good stuff 🐕

**Comment by SiAdcock** [comment]
Have you tested this in a screen reader / using keyboard only?

**Comment by walaura** [comment]
Thanks! Fyi this is on hold from being merged until we get the definite go-ahead from the rest of stakeholders, which should happen at some point tomorrow. 🐶

**Comment by SiAdcock** [comment]
Aw, thank you! 😍

File: common/app/views/fragments/inlineSvg.scala.html
Comment (walaura) [inline common/app/views/fragments/inlineSvg.scala.html]: @SiAdcock reviewing myself here. What if instead of `isPresentation: Boolean` I made it `description: Option[String] = None`? 

If there's a description it uses it as alt text, if there's nothing it puts `aria-hidden` as it is now

I don't think any of our svgs contain alt text inline (probably shouldn't) so I don't think there's any point in making screen readers try to digest them unless they are explicitly described?

**Review by rahilb — APPROVED** [review]

File: common/app/views/fragments/inlineSvg.scala.html
Comment (SiAdcock) [reply common/app/views/fragments/inlineSvg.scala.html]: Interesting. Can we make it `title`, and use it to add a [`<title>` element](https://css-tricks.com/accessible-svgs/#article-header-id-9) to the SVG?

I'm somewhat uncomfortable adding `aria-hidden="true"` to every existing inline SVG without assessing the impact. What would happen to SVGs used as links? Would screen readers not see the link at all?

I would rather exercise caution and keep the `isPresentation: Boolean = false` parameter, rather than tying `aria-hidden="true"` to the existence of a `title`.

Comment (walaura) [reply common/app/views/fragments/inlineSvg.scala.html]: okay!

**Pull request merged**
